modules/oe_whitelabel_search/oe_whitelabel_search.post_update.php

```diff
@@ -45,3 +45,18 @@ function oe_whitelabel_search_post_update_00002(&$sandbox) {
   }
   return 'No search block needed to be updated.';
 }
+
+/**
+ * Move the Whitelabel Search Block to the 'header_top' region.
+ */
+function oe_whitelabel_search_post_update_00004(&$sandbox) {
+  $block = Block::load('oe_whitelabel_search_form');
+  if (!$block || $block->getTheme() !== 'oe_whitelabel') {
+    return 'No update needed.';
+  }
+  if ($block->getRegion() !== 'header_top') {
```

Comment:
I would say we should only move the block if it is currently in 'navigation_right', ~(and if it is enabled)~.
If it is in another region or disabled, that means the site builder does not want the default behavior.

> ~(and if it is enabled)~

If the block is simply disabled, I think we can still move it, but keep it disabled.

Reply:
Also we can swap the condition so that we have symmetry with the earlier condition.
```suggestion
  if ($block->getRegion() === 'header_top') {
    return 'Search block already in correct region.';
  }
  if ($block->getRegion() !== 'navigation_right') {
    return 'The search block was intentionally moved to a different region.';
  }
  $block->setRegion('header_top')->save();
  return "Search block moved to 'header_top'.";
}
```

Reply:
changed